src/vl-map-draw-point-action.js

```diff
@@ -9,13 +9,16 @@ import {VlDrawAction, OlGeometryType} from '/node_modules/vl-mapactions/dist/vl-
  *
  * @extends VlMapDrawAction
  *
+ * @property {string} [data-vl-snapping] - Attribuut wordt gebruikt om aan te geven dat er bij het tekenen snapping mag gebeuren, hetzij op de laag waarop getekend wordt (indien geen vl-map-wfs-layer(s) als child elementen), hetzij op de meegegeven vl-map-wfs-layers.
+ * @property {number} [data-vl-snapping-pixel-tolerance=10] - Attribuut om aan te geven binnen de hoeveel pixel van een feature er gesnapped mag worden.
+ *
```

Comment:
Best verhuizen naar `vl-map-action` aangezien het op dat niveau nu van toepassing is.